src/FluentResults/Settings/ResultSettings.cs

```diff
@@ -1,8 +1,6 @@
-﻿// ReSharper disable once CheckNamespace
+﻿using System;
 
-using System;
-
-namespace FluentResults
+namespace FluentResults.Settings
```

Comment:
keep it in the FluentResults namespace. This would be a breaking change for clients

Reply:
Yes, but I think I would like to propose some refactoring in some namespaces then in the future.